README.rst

```diff
@@ -168,6 +168,28 @@ information, please read the
         }
     }
 
+
```

Comment:
Now we can put the gevent extension link in this section: https://github.com/census-instrumentation/opencensus-python#extensions.

We need to add it in the test script https://github.com/census-instrumentation/opencensus-python/blob/master/nox.py#L33, and have test cases.

Reply:
FYI, I just updated the CONTRIBUTING document:

https://github.com/census-instrumentation/opencensus-python/blob/master/CONTRIBUTING.md#additional-instructions-for-extension-developers

Reply:
Added a test which triggers the gevent patching and checks that the context is switched as expected.